Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,8 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+
+.app-container {
+  width: 40%;
```

Comment:
I presume you arrived at this number based on a wide viewport. It doesn't work when the viewport is smaller. Set `width: 100%` and then set `max-width: 500px` or something in order to provide a better UI.